Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -243,6 +243,135 @@ s5cmd --endpoint-url http://$AWS_HOST:$PORT cp s3://$BUCKET_NAME/rookObj /tmp/ro
 cat /tmp/rookObj-download
 ```
 
+### Create a Bucket via Ceph COSI Driver
+
+The Ceph COSI driver will be started automatically with default settings when first Object Store gets created. The driver will be deleted when Rook Operator is uninstalled. The driver will be deployed in the same namespace as Rook operator. The COSI controller needs already running if the Ceph COSI Driver needs to come up. The COSI controller can be deployed by following the commands below:
+
+```bash
+kubectl apply -k github.com/kubernetes-sigs/container-object-storage-interface-api ## installs CRDs
```

Comment:
Can we include them in the helm chart? Or is there a COSI helm chart that we could make a dependency? For experimental mode let's not worry about the helm chart, but when COSI is considered stable we need it to be considered via helm.

Reply:
I send out to PR to create a helm chart for the cosi controller, it's not review till now https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/75. Even I like to do it via helm instead of using kubectl